provider/source/data/debug/time_zone/iana_to_bcp47@2.json

```diff
@@ -385,216 +384,222 @@
     "Pacific/Easter": 225,
-    "Pacific/Efate": 887,
-    "Pacific/Enderbury": 414,
-    "Pacific/Fakaofo": 725,
-    "Pacific/Fiji": 295,
-    "Pacific/Funafuti": 739,
-    "Pacific/Galapagos": 269,
-    "Pacific/Gambier": 575,
-    "Pacific/Guadalcanal": 673,
-    "Pacific/Guam": 355,
-    "Pacific/Honolulu": 771,
-    "Pacific/Johnston": 770,
-    "Pacific/Kanton": 415,
-    "Pacific/Kiritimati": 413,
-    "Pacific/Kosrae": 299,
-    "Pacific/Kwajalein": 477,
-    "Pacific/Majuro": 479,
-    "Pacific/Marquesas": 577,
-    "Pacific/Midway": 753,
-    "Pacific/Nauru": 561,
-    "Pacific/Niue": 563,
-    "Pacific/Norfolk": 549,
-    "Pacific/Noumea": 545,
+    "Pacific/Efate": 879,
+    "Pacific/Enderbury": 410,
+    "Pacific/Fakaofo": 717,
+    "Pacific/Fiji": 291,
+    "Pacific/Funafuti": 731,
+    "Pacific/Galapagos": 267,
+    "Pacific/Gambier": 569,
+    "Pacific/Guadalcanal": 665,
+    "Pacific/Guam": 351,
+    "Pacific/Honolulu": 763,
+    "Pacific/Johnston": 762,
+    "Pacific/Kanton": 411,
+    "Pacific/Kiritimati": 409,
+    "Pacific/Kosrae": 295,
+    "Pacific/Kwajalein": 473,
+    "Pacific/Majuro": 475,
+    "Pacific/Marquesas": 571,
+    "Pacific/Midway": 745,
+    "Pacific/Nauru": 555,
+    "Pacific/Niue": 557,
+    "Pacific/Norfolk": 543,
+    "Pacific/Noumea": 539,
     "Pacific/Pago_Pago": 63,
-    "Pacific/Palau": 605,
-    "Pacific/Pitcairn": 593,
-    "Pacific/Pohnpei": 301,
-    "Pacific/Ponape": 300,
-    "Pacific/Port_Moresby": 581,
+    "Pacific/Palau": 597,
+    "Pacific/Pitcairn": 587,
+    "Pacific/Pohnpei": 297,
+    "Pacific/Ponape": 296,
+    "Pacific/Port_Moresby": 575,
     "Pacific/Rarotonga": 223,
-    "Pacific/Saipan": 495,
+    "Pacific/Saipan": 491,
     "Pacific/Samoa": 62,
-    "Pacific/Tahiti": 579,
-    "Pacific/Tarawa": 417,
-    "Pacific/Tongatapu": 733,
-    "Pacific/Truk": 302,
-    "Pacific/Wake": 751,
-    "Pacific/Wallis": 889,
-    "Pacific/Yap": 302,
-    "Poland": 588,
-    "Portugal": 600,
-    "ROC": 740,
-    "ROK": 424,
-    "Singapore": 680,
-    "Turkey": 734,
-    "UCT": 814,
-    "US/Alaska": 760,
-    "US/Aleutian": 756,
-    "US/Arizona": 800,
-    "US/Central": 764,
-    "US/East-Indiana": 772,
-    "US/Eastern": 794,
-    "US/Hawaii": 770,
-    "US/Indiana-Starke": 778,
-    "US/Michigan": 768,
-    "US/Mountain": 766,
-    "US/Pacific": 780,
-    "US/Pacific-New": 780,
+    "Pacific/Tahiti": 573,
+    "Pacific/Tarawa": 413,
+    "Pacific/Tongatapu": 725,
+    "Pacific/Truk": 298,
+    "Pacific/Wake": 743,
+    "Pacific/Wallis": 881,
+    "Pacific/Yap": 298,
+    "US/Alaska": 752,
+    "US/Aleutian": 748,
+    "US/Arizona": 792,
+    "US/Central": 756,
+    "US/East-Indiana": 764,
+    "US/Eastern": 786,
+    "US/Hawaii": 762,
+    "US/Indiana-Starke": 770,
+    "US/Michigan": 760,
+    "US/Mountain": 758,
+    "US/Pacific": 772,
+    "US/Pacific-New": 772,
     "US/Samoa": 62,
-    "UTC": 814,
-    "Universal": 814,
-    "W-SU": 638,
-    "Zulu": 814
+    "_CET": 100,
```

Comment:
Observation: None of these `_` names are canonical. I can tell because they are all even numbers.